src/components/IconButton.tsx

```diff
@@ -1,6 +1,7 @@
-import styled from "@emotion/native"
 import React from "react"
+import { StyleSheet, TouchableOpacity } from "react-native"
```

Comment:
I guess this answers my question https://github.com/Path-Check/gaen-mobile/pull/26#discussion_r458072265 😅  I'll update the tests once you merge this one